timApp/velp/velp.py

```diff
@@ -711,7 +711,21 @@ def create_velp_group_route(doc_id: int) -> dict:
             new_group_path
         )  # Check name so no duplicates are made
         if group_exists is None:
-            original_owner = target.owners[0]
+            # Document may not have an owner, we have to account for that
+            if not len(target.owners) < 1:
```

Comment:
Vähän "pythonimaisempi" tapa tarkistaa, onko lista epätyhjä:

```suggestion
            if target.owners:
```